scalameta/contrib/src/test/scala/scala/meta/contrib/ScaladocParserSuite.scala

```diff
@@ -51,6 +51,31 @@ class ScaladocParserSuite extends FunSuite {
     assert(
       parseString(
         s"""
+         // $expectedBody
```

Comment:
Shouldn't all comments that start with `//` or `/*`  be unparsable by the scaladoc parser? That would be my expected behavior. Since they are not scaladocs.